src/lib/rustContext.ts

```diff
@@ -56,7 +57,7 @@ export default class RustContext {
     // We need this await here, DO NOT REMOVE it even if your editor says it's
     // unnecessary. The constructor of the module is async and it will not
     // resolve if you don't await it.
```

Comment:
Should we change the comment here?

Reply:
NO we need to listen to the comment hahaha